gateway-service-impl/src/test/java/org/hypertrace/gateway/service/common/datafetcher/EntityDataServiceEntityFetcherTests.java

```diff
@@ -187,6 +190,50 @@ public void test_getTimeAggregatedMetrics() {
         });
   }
 
+  @Nested
```

Comment:
Why do we need this? Why not just have TotalEntitiesTest class?

Reply:
The method `getTotal` lies in the same class as `EntityDataServiceEntityFetcher`

Nested annotation makes the unit test more beautiful and structured, such that any unit tests related to total would go inside this Nested class, and won't be spread and mixed up across unit tests